doc/rst/technotes/gpu.rst

```diff
@@ -149,10 +147,15 @@ The following are further requirements for GPU support:
     * You can check the current status of ROCm version support `here
       <https://github.com/chapel-lang/chapel/issues/23480>`_.
 
-  * ``CHPL_LLVM`` must be set to ``system``. Note that, ROCm installations come
-    with LLVM. Setting ``CHPL_LLVM=system`` will allow you to use that LLVM.
+  * For ROCm 5.x, ``CHPL_LLVM`` must be set to ``system``. Note that, ROCm
+    installations come with LLVM. Setting ``CHPL_LLVM=system`` will allow you to
+    use that LLVM.
```

Comment:
Should `CHPL_LLVM` be `system` or `bundled` otherwise? If using NVIDIA or ROCM < 5.x?

Reply:
Yes. Exactly.